jetstream_pt/config.py

```diff
@@ -139,6 +139,18 @@
     "size of top k used when sampling next token",
 )
 
+flags.DEFINE_integer(
+    "total_num_pages",
```

Comment:
rename to pagedattention_total_num_pages

Reply:
Changed.